bedrock/base/templates/includes/protocol/navigation/menus-refresh/about-us.html

```diff
@@ -9,51 +9,80 @@
-                <h4 class="m24-c-menu-item-title">{{ ftl('navigation-refresh-blog') }}</h4>
-              </a>
-            </section>
-          </li>
-         </ul>
+       <div class="m24-c-menu-panel-content multi-column">
+        <div class="m24-mzp-l-content-container">
+          <p>{{ ftl('navigation-refresh-our-mission') }}</p>
+          <ul class="m24-mzp-l-content">
+            <li>
+              <section class="m24-c-menu-item">
+                <a class="m24-c-menu-item-link" href="{{ url('mozorg.about.index') }}" data-link-text="About Mozilla" data-link-position="topnav - about">
+                  <h4 class="m24-c-menu-item-title">{{ ftl('navigation-refresh-about-mozilla') }}</h4>
+                </a>
+              </section>
+            </li>
+            <li>
+              <section class="m24-c-menu-item">
+                <a class="m24-c-menu-item-link" href="{{ url('mozorg.about.manifesto') }}" data-link-text="The Mozilla manifesto" data-link-position="topnav - about">
+                  <h4 class="m24-c-menu-item-title">{{ ftl('navigation-refresh-mozilla-manifesto-v2', fallback='navigation-refresh-mozilla-manifesto') }}</h4>
+                </a>
+              </section>
+            </li>
+            <li>
+              <section class="m24-c-menu-item">
+                <a class="m24-c-menu-item-link" href="{{ url('mozorg.contribute') }}" data-link-text="Get Involved" data-link-position="topnav - about">
+                  <h4 class="m24-c-menu-item-title">{{ ftl('navigation-refresh-get-involved-v2', fallback='navigation-refresh-get-involved') }}</h4>
+                </a>
+              </section>
+            </li>
+            <li>
+              <section class="m24-c-menu-item">
+                <a class="m24-c-menu-item-link" href="https://future.mozilla.org/?{{ utm_params }}" data-link-text="Innovation Projects" data-link-position="topnav - about">
+                  <h4 class="m24-c-menu-item-title">{{ ftl('navigation-refresh-innovation-projects-v2', fallback='navigation-refresh-innovation-projects') }}</h4>
+                </a>
+              </section>
+            </li>
+            <li>
+              <section class="m24-c-menu-item">
+                <a class="m24-c-menu-item-link" href="https://blog.mozilla.org/?{{ utm_params }}" data-link-text="Blog" data-link-position="topnav - about">
+                  <h4 class="m24-c-menu-item-title">{{ ftl('navigation-refresh-blog') }}</h4>
+                </a>
+              </section>
+            </li>
+           </ul>
+        </div>
+        <div class="m24-mzp-l-content-container">
+          <p>{{ ftl('navigation-refresh-our-work') }}</p>
+          <ul class="m24-mzp-l-content">
+            <li>
+              <section class="m24-c-menu-item">
+                <a class="m24-c-menu-item-link" href="https://foundation.mozilla.org/?{{ utm_params }}" data-link-text="Mozilla Foundation" data-link-position="topnav - about">
```

Comment:
**issue (blocking):** sorry i missed this at first, but it looks like `utm_params` is undefined for this menu, so it's instead inheriting it from the home page template rendering an incorrect URL.

I think you need to add the following line to the top of this file for it to work correctly?

```
{% set utm_params = 'utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=nav&utm_content=about-us' %}
```